pypowerwall/__init__.py

```diff
@@ -475,6 +511,45 @@ def get_reserve(self, scale=True) -> Optional[float]:
             return percent
         return None
 
+    def get_mode(self, force=False) -> Optional[float]:
+        """
+        Get Battery Operation Mode
+        """
+        data = self.poll('/api/operation', force=force)
+        if data and data.get('real_mode'):
+            return data['real_mode']
+        return None
+
+    def set_battery_op_reserve(self, level: Optional[float] = None, mode: Optional[str] = None,
```

Comment:
Open to suggestions if the name of the function is too long/odd.

Reply:
It is fine, but we could abbreviate as `set_operation()` since that is the Tesla API and it describes what it does, but I would like to see two interfaces to this that align to their "get" equivalents:

```
set_reserve(level)
set_mode(mode)
```